### PR TITLE
Devx 634 - Add document to describe how bundling works

### PR DESCRIPTION
**Pull request by dpgraham**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/bundling.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+id: bundling
+title: Bundling Test Files
+sidebar_label: CLI Reference
+---
+
+The [saucectl command line interface](saucectl) runs tests (Cypress, TestCafe, Playwright, ...) by bundling up your root directory, sending the bundle to the Sauce Labs cloud (or a Docker container), unpacking the bundle and then running the tests. This page explains how bundling works and how you can reduce the size of your bundle for improved performance.
```
Comment (KunalJain) [inline docs/testrunner-toolkit/bundling.md:7]: Might be good to call out that the Docker container is for your local or your own infra.

File: docs/testrunner-toolkit/bundling.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+sidebar_label: CLI Reference
+---
+
+The [saucectl command line interface](saucectl) runs tests (Cypress, TestCafe, Playwright, ...) by bundling up your root directory, sending the bundle to the Sauce Labs cloud (or in your own infrastructure via Docker), unpacking the bundle and then running the tests. This page explains how bundling works and how you can reduce the size of your bundle for improved performance.
+
+## Bundling
+
+The Sauce Labs config.yml has a parameter called "rootDir" that specifies the directory that needs to be bundled ([see configuration documentation](configuration.md))
+
+(TODO: add a screengrab of the bundling in action)
```
Comment (alexplischke) [inline docs/testrunner-toolkit/bundling.md:13]: Does that show up? :D

Comment (dpgraham) [reply docs/testrunner-toolkit/bundling.md]: Adde dit @alexplischke

**Pull request merged**